gcm/connector.go

```diff
@@ -126,7 +126,7 @@ func (conn *Connector) loopPipeline(id int) {
 }
 
 func (conn *Connector) sendMessage(pm *pipeMessage) {
-	gcmID := pm.subscription.route.ApplicationID
+	gcmID := pm.subscription.route.Get("application_id")
```

Comment:
define these strings as constants, use them all over the code:
* "application_id"
* "user_id"